### PR TITLE
RightSidebarPartsView

### PR DESCRIPTION
**Pull request by thwalker6**
Resolves # EREGCSC-1185

**Description-**

Right sidebar view for displaying mulitple supplmenetal content resources.
**This pull request changes...**

- By navigating to the content you can see a sidebar with various dropdowns to select supplemental content.  Different tabs for parts, subparts, and sections.   Some functionality is yet to be added becasue we do not have the information needed nor do they do they know what they want.

**Steps to manually verify this change...**

1. steps to view and verify change

### CODE REVIEW

File: solution/ui/prototype/src/components/PDPart/PartSummary.vue [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+                    >
+                </v-expansion-panel>
+            </v-expansion-panels>
+        </div>
+        <br />
+        <v-btn color="#046791" class="white--text">View All Resources</v-btn>
+    </div>
+</template>
+<script>
+import RelatedRules from "../../../../regulations/js/src/components/RelatedRules.vue";
```
Comment (knollfear) [inline solution/ui/prototype/src/components/PDPart/PartSummary.vue:75]: Phil aliased in the existing site with this:
import RelatedRules from "legacy/js/src/components/RelatedRules.vue";

File: solution/ui/prototype/src/components/PDPart/SectionCards.vue [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+                                    show ? "mdi-chevron-up" : "mdi-chevron-down"
+                                }}</v-icon>
+                            </v-btn>
+                        </v-card-actions>
+
+                        <v-expand-transition>
+                            <div v-show="show">
+                                <v-divider></v-divider>
+
+                                <v-card-text>
```
Comment (knollfear) [inline solution/ui/prototype/src/components/PDPart/SectionCards.vue:33]: Is this futurama ipsum?

File: solution/ui/prototype/src/components/PDPart/Subpart.vue [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                        v-bind:subpart="subpart" /></v-expansion-panel-content
+            ></v-expansion-panel>
+        </v-expansion-panels>
+        <br />
+    </div>
+</template>
+<script>
+
+import { getSupplementalContentNew, getSubPartsForPart } from "@/utilities/api";
+import SupplementalContentCategory from "../../../../regulations/js/src/components/SupplementalContentCategory.vue";
```
Comment (knollfear) [inline solution/ui/prototype/src/components/PDPart/Subpart.vue:35]: another legacy import

File: solution/ui/prototype/src/utilities/api.js [hunk 4]
```diff
@@ -431,14 +431,35 @@ const getSupplementalContent = async (
...
+    title,
+    part,
+    sections =[],
+    subparts= []
+) => {
+    let sString = '';
+    for (let s in sections) {
+        sString = sString + "&sections=" + sections[s]
+    }
+    console.log(sString)
```
Comment (knollfear) [inline solution/ui/prototype/src/utilities/api.js:452]: probably unnecessary

File: solution/ui/prototype/src/utilities/api.js [hunk 5]
```diff
@@ -431,14 +431,35 @@ const getSupplementalContent = async (
...
+    for(let sp in subparts){
+        sString = sString + "&subparts=" + subparts[sp]
+    }
+    console.log(sString)
```
Comment (knollfear) [inline solution/ui/prototype/src/utilities/api.js:456]: probably unnecessary

File: solution/ui/prototype/src/components/PDPart/SectionCards.vue (hunk 2 above)
Comment (thwalker6) [reply solution/ui/prototype/src/components/PDPart/SectionCards.vue:33]: Yup....

File: solution/ui/prototype/src/utilities/api.js (hunk 5 above)
Comment (thwalker6) [reply solution/ui/prototype/src/utilities/api.js:456]: Removed

**Pull request merged**